tests/test_opendss_transformer.py

```diff
@@ -27,13 +27,15 @@ def test_opendss_center_transformer():
     ~ conns=[delta wye wye]
     """
 
-    master_file = tempfile.NamedTemporaryFile(mode="w")
+    master_file = os.path.join(tempfile.gettempdir(), os.urandom(24).hex())
+    # TODO: figure out why named temporary file doesn't work as expected on windows
+    # master_file = tempfile.NamedTemporaryFile(mode="w")
```

Comment:
The documentation seems to say that what you are doing isn't allowed on Windows (1. Can't have two open file handles, 2. If the file is closed, it will be deleted).  What about using `tempfile.mkstemp()` to create a file, close it immediately, and then continue as you are doing now?

Also, you aren't deleting the file currently.  Maybe that OK as it's in /tmp.  I'm guessing you wanted that automatic functionality with NamedTemporaryFile.

Reply:
What you suggested is better. I'll do that in a separate PR.